lesson02/assignment/css/_components.content.scss

```diff
@@ -1,17 +1,180 @@
 html {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 *,
 *:after,
 *:before {
-    box-sizing: inherit;
+  box-sizing: inherit;
 }
 
 .main {
+  padding: 2rem;
+}
+// ========TYPE STYLES========
+h1 {
+  font-size: 1.75rem;
+  margin-top: 2rem;
+}
+
+h1,
+h2,
+h3,
+h4,
+span {
+  color: $charcoal;
+}
+
+h3,
+h4 {
+  display: inline;
+  line-height: 2.5rem;
+}
+
+h2,
+h4 {
+  font-weight: 300;
+}
+
+h2 {
+  font-size: 1.25rem;
+  font-style: italic;
+  line-height: 0;
+}
+// ========FLEX BOX========
+.card {
+  &__container {
+    background-color: $gray;
+    border: 1px solid $slate;
+    border-radius: 5px;
+    padding: 3rem;
+  }
+
+  &__entry {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    background-color: $white;
+    padding: 2.5rem;
+  }
+
+  &__identity {
+    padding: 0;
+    margin: 0 2rem 0 0;
+    text-align: center;
+
+    & img {
+      border-radius: 50%;
+      width: auto;
+      max-width: 220px;
+      min-width: 200px;
```

Comment:
For maximum responsivity, almost all sizing should be in rems or %. Exceptions are things like borders, and sometimes icons (LinkedIn icon below being sized with px is fine). If, for example, this card was being viewed on a viewscreen that was 300px wide, the image being a fixed 220px wide would be a problem.

Reply:
Thanks for the explanation! I've made the changes to rems in a new [PR](https://github.com/Steve-Carlton/html300/pull/3). It works perfectly!